### PR TITLE
Add UpdateSourcesForUserCmd to example-Cli

### PR DESCRIPTION
**Pull request by Sriep**

### CODE REVIEW

File: builtin/types/karma/karma.proto [hunk 1]
```diff
@@ -5,33 +5,27 @@ import "github.com/loomnetwork/go-loom/types/types.proto";
...
     int64 max_karma = 1;
-    Validator Validator = 2;
+    Address oracle = 2;
 }
 
 message KarmaParamsValidatorNewOracle {
-    Address NewOracle = 1;
-    Validator Validator = 2;
+    Address new_oracle = 1;
+    Address old_oracle = 2;
```
Comment (mattkanwisher) [inline builtin/types/karma/karma.proto]: why do you need old oracle on an update? is this just being stored into the state db?

**Comment by mattkanwisher** [comment]
one minor comment otherwise you are good to go

File: builtin/types/karma/karma.proto (hunk 1 above)
Comment (Sriep) [reply builtin/types/karma/karma.proto]: The idea is that only the old/current Oracle can change the Oracle. I did it this way to reuse existing code with validator replaced by Oracle.

Comment (Sriep) [reply builtin/types/karma/karma.proto]: On inspection, it looks like old oracles are kept. There is an error message that distinguishes between an old-oracle or an invalid one.

**Pull request merged**